znoyder/templater.py

```diff
@@ -58,6 +61,20 @@ def generate_zuul_config(path: str, name: str,
 
     jobs_dict = defaultdict(list)
 
+    try:
+        JOB_TEMPLATE = j2env.get_template(template_name+".j2")
+    except TemplateNotFound:
+        script_dir = os.path.dirname(os.path.realpath(__file__))
+
+        LOG.error(f'Template "{template_name}" does not exist')
+        LOG.info('Please use one of the following templates or add your own')
+        LOG.info('Available templates:')
+        for template in os.listdir(f'{script_dir}/templates'):
```

Comment:
Maybe we could use the jinja environment for this? After a quick look in their docs I found the [list_templates method](https://jinja.palletsprojects.com/en/3.0.x/api/#jinja2.Environment.list_templates) that seems to do the same. But I have no experience with jinja, so I'm not sure

Reply:
I did try to use the list_templates method to no avail.

Reply:
It seems to be working for me, but the current method does not. The call to isfile fails because listdir does not return an absolute path, just the name of the file. Passing the name to isfile will fail unless running from inside the templates folder

Reply:
Yup, failing on me as well.
I got a fix coming up using the get_template() method from jinja2 that fixes the issue.